src/components/atoms/Link/index.jsx

```diff
@@ -0,0 +1,3 @@
+import Link from './Link';
```

Comment:
suggestion: We can reduce the two line to: 

```suggestion
export { default } from './Link';
```